project_proposal/front_page.tex

```diff
@@ -0,0 +1,47 @@
+\begin{titlepage}
+\begin{center}
+% Title
+{ \LARGE \bfseries Bierproductie \\[0.4cm]}
+A management system for brewing machines
+\begin{figure}[H]
+\centering 
+\includegraphics[scale=0.5]{figures/creditoro.png}

```

Comment:
Missing a frontpage figure.

Reply:
Yup, wasn't sure what figure to go with.

Reply:
Maybe we could go with the one from project proposal made by @nymann

Reply:
I've added the system drawing